python/raydp/tests/test_spark_cluster.py

```diff
@@ -61,6 +61,12 @@ def test_spark_driver_and_executor_hostname(spark_on_ray_small):
     driver_bind_address = conf.get("spark.driver.bindAddress")
     assert node_ip_address == driver_bind_address
 
+def test_ray_dataset_from_and_to_spark(spark_on_ray_small):
+    spark = spark_on_ray_small
+    words_df = spark.createDataFrame([('look',), ('spark',), ('tutorial',), ('spark',), ('look', ), ('python', )], ['word'])
+    ds = raydp.spark.spark_dataframe_to_ray_dataset(words_df)
+    df = raydp.spark.ray_dataset_to_spark_dataframe(spark, ds)
+    assert words_df.toPandas().equals(df.toPandas())
```

Comment:
Can we add a corresponding unit test on the Ray side that it isn't breaking raydp?